samples-js/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "samples-js",
+    "version": "1.0.0",
+    "description": "",
+    "main": "functions/*.js",
+    "scripts": {
+        "start": "func start",
+        "test": "echo \"No tests yet...\""
+    },
+    "dependencies": {
+        "@azure/functions": "^4.0.0-alpha.6",
+        "@azure/storage-blob": "^12.12.0",
+        "axios": "^1.2.3",
+        "durable-functions": "^3.0.0-alpha.1",
+        "luxon": "^3.2.1",
+        "readdirp": "^3.6.0"
+    },
+    "devDependencies": {
+        "chai": "^4.3.7",
+        "mocha": "^10.2.0",
+        "sinon": "^15.0.1"
```

Comment:
do we really need these given that we don't have unit tests?

Reply:
No we don't lol, I added them when I was trying to add unit testing samples and forgot to remove them when I failed

Reply:
Removed